README.md

```diff
@@ -101,7 +104,7 @@ so that the migrations will be run in the pipelines to correctly upgrade the dep
 
 ## CLI Commands
 
-These commands can be run from the command-line from a terminal using the same python environment as a running flask application.
+If running the data-store app locally using Flask (i.e not through the Docker runner), these commands can be run from the command-line from a terminal using the same python environment as the running flask application.
```

Comment:
I think this could currently be a bit misleading, as you can still run these commands if it's running in Docker - you'd just have to use the containers console